react/week4/github-search-user/src/Components/CreateContext.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { createContext, useState } from "react";
+
+// @ts-ignore
+export const CreatingUserContext = createContext();
+
+const GithubSearchContext = ({ children }) => {
+  const [input, setInput] = useState("");
```

Comment:
You can put these states in a Context.js file and consume them here.

Reply:
Hej @annamourasobral, Thank you so much for the comments on my code ...